pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -452,7 +452,7 @@ func validateName(name string) error {
 			return fmt.Errorf("invalid resource property name %s, expected format is [PREFIX]/[CAPACITY_TYPE]-[RESOURCE_NAME]", name)
 		}
 		if !supportedResourceCapacityTypesMap[segments[0]] {
-			return fmt.Errorf("invalid capacity type in resource property name %s, supported values are %+v", name, supportedResourceCapacityTypes)
+			return fmt.Errorf("invalid capacity type in resource property name %s, supported values are %+v", name, resourceCapacityTypes)
```

Comment:
would  supportedResourceCapacityTypes() work here?

Reply:
Map was suggested to make the lookup faster in this comment https://github.com/Azure/fleet/pull/808#discussion_r1605199781